front/src/modules/pathfinding/components/Pathfinding/PathfindingV2.tsx

```diff
@@ -318,8 +318,13 @@ const Pathfinding = ({ pathProperties, setPathProperties }: PathfindingProps) =>
                       kp: correspondingOp.kp,
                       coordinates: correspondingOp.coordinates,
                       positionOnPath: pathfindingResult.path_items_positions[i],
+                      stopFor: i === pathSteps.length - 1 && !step.stopFor ? '0' : step.stopFor,
```

Comment:
Store this instruction inside a variable stopFor and reuse it line 321 and 326
`const stopFor = i === pathSteps.length - 1 && !step.stopFor ? '0' : step.stopFor;`
```suggestion
                      stopFor,
```

Reply:
Done